web/src/pages/about.js

```diff
@@ -21,6 +21,22 @@ export const query = graphql`
                 }
             }
             pageTitle
+            aboutSection {
+                _rawLeftColumn
+                _rawRightColumn
+            }
+            leadershipSection {
+                leadership {
+                    name
+                    title
+                    image {
+                        asset {
+                            gatsbyImageData
+                        }
+                    }
+                    linkedinURL
```

Comment:
I don't see where this is used in the about page.

Reply:
in the leadership card @asasmith

Reply:
I don't see that in the designs

Reply:
@asasmith If I remember correctly the cards are to link to their LinkedIn, but i am confirming